config/flags/flags.go

```diff
@@ -365,4 +365,10 @@ var (
 			"Setting this value to <=2000 will result in pre-confugired cache capacity of 2KB", CacheFlag.Name),
 		Value: 0,
 	}
+	StateDbCacheCapacityFlag = cli.IntFlag{
+		Name: "statedb.cache",
+		Usage: "Size of StateDb instances cache in bytes. Leaving this blank (which is generally recommended), or setting" +
```

Comment:
This cache parameter is actually not in bytes but in number of elements in the storage data cache created [here](https://github.com/Fantom-foundation/Carmen/blob/690bc10fa6240be832fbc8a93fdcf6023350b4a1/go/state/state_db.go#L457). Each of those entries has a considerable size (approximately 20+32 bytes for the slot ID and 32+8 for the value, assuming no overhead in the LRU cache). So each entry takes ~92 bytes. 

We could document this in the flag, or do a conversion from the flag's value to the value forwarded to the implementation. This way, the flag could be in bytes. 

But since this flag is not really expected to be set by the user, just by internal tests and tools, documenting it as an "experimental" feature and updating it to say that it is the number of cached data elements is probably the simpler and less error-prone way to go. Wdyt?

Reply:
I thought the unit would be incorrect. I think adding saying that its 'the number of cached data elements' is good. I will also add a suggestion that for tests value <1024 is not recommended.